RELEASE.md

```diff
@@ -0,0 +1,101 @@
+
+#### Ensure release branch includes all relevant fixes
+- Make release branch if needed (major/minor).
+  - For fix releases, the branch should already exist.
+  - Otherwise create branch: `release/X.X`.
+- Update your local checkout to tip of the release branch.
+- Cherry pick all PRs into the release branch:
+  - If branch is already labelled `status: cherry-picked 🍒` then continue to next PR.
+  - Ideally, use GitHub Hub to cherry pick the PR - `hub am -3 {http://URL-TO-PR}`. 
+  - If there are serious conflicts or extensive differences between `master` and release branch, you may need to take more care:
+    - Manually cherry pick individual commits using git - `git cherry-pick {COMMIT-HASH}`.
+    - Or in some cases, manually craft a new PR with appropriate changes, targeting release branch.
+  - Push the release branch to origin (so changes are in GitHub repo).
+  - Label the PR: `status: cherry-picked 🍒`.
+
+_Outcome_: __Release branch has all relevant changes merged & pushed.__
+
+### Prepare release
+#### Ensure release branch readme is up to date
+- Run changelog script `npm run changelog` to get changelog txt for readme. Changelog content will be output to screen by script.
+- Add changelog section for release, e.g. `= 2.5.11 - 2020-01-20 =`.
+- Copy-paste the changelog content into `readme.txt`.
+- Make any other changes to readme as needed - e.g. support versions changing, new blocks. 
+- Push readme changes to release branch on origin repo.
+
+_Outcome_: __`readme.txt` is up to date with changes for release.__
+
+#### Build zip & smoke test
+- Ensure you are on tip of release branch.
+- Update dependencies – `npm ci`.
+- Run a production build - `npm run build`.
+- Run package script to get a zip to test `npm run package-plugin`.
+- Smoke test built release zip:
+  - Recommend doing this in a clean environment, e.g. Jurassic.Ninja site.
+  - Sanity check - check correct blocks exist and function.
+  - Test to confirm new features/fixes are working correctly.
+  - Smoke test – test a cross section of core functionality.
+
+_Outcome_: __Confident that source code is ready for release: intended fixes are working correctly, no release blockers or build issues.__
+
+### Release!
+#### Release to GitHub
+- Prepare tagged release on github `npm run deploy`. 
+  - Note: the script automatically updates version numbers (commits on your behalf).
+- Edit release, add changelog info to Github release notes.
+- Check release repo tag is correct - checkout, smoke test/sanity check.
+
+_Outcomes_: __Version numbers updated in source code & developers can test tagged release.__
+
+##### Release to WPORG
+- Push release to WPORG `npm run release`.
+- Note that this may happen a day or so after Github release.
+- FYI: Sometimes we make small changes (e.g. readme only) by [committing directly to svn repo](https://developer.wordpress.org/plugins/wordpress-org/how-to-use-subversion/#editing-existing-files).
```

Comment:
It's quite similar to the page you linked, but in `CONTRIBUTING.md` we have these steps detailed as well:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/CONTRIBUTING.md#updating-a-single-file-in-wordpressorg
The nice thing is that commands are already with our repo URL, so it's just copy & pasting into the terminal.

Do you think it might be worth keeping it or you removed it in purpose?

Reply:
> Do you think it might be worth keeping it or you removed it in purpose?

I agree with Albert here, it'd be nice to keep the example commands for updating the readme.txt here.

Reply:
> Do you think it might be worth keeping it or you removed it in purpose?

Yes, I did remove this on purpose :) To me it didn't seem essential for us to document this – it's just committing a change to a file (we're not special). The key thing was to note that this sometimes happens and is part of how we do things. Also it's a little distracting having this in the release process.

So .. I've reinstated the previous doc (I don't think it hurts to include it) as an "appendix" on the release page. 3ba987f0a513575ab2a7fec82bef145602201ead